### ISSUE TITLE
Localization plural problems

### ISSUE THREAD

**Issue by slavkoja** [keepassxreboot/keepassxc#2913]
I am not sure where to fill it, but in recent time i have troubles with plurals translation (Slovak, sk_SK language) via Transifex, which ended with unsaved translations.

## Expected Behavior
The string with singular and plural have to be without alternatives, eg. string for singular have to be "%d min" (expanded to "1 min") and string for plural "%s mins" (expanded to "5 mins"). 

In our language we have two forms of plurals, which ends in three forms -- singular, plural for 2- 4 items and plural for 5 and more items. By this, the parentheses are unusable for good translation, because they are useful only for two forms languages.

## Current Behavior
Not in all, but in a lot of messages i have troubles with these plurals, because the original for both, singular and plural are the same, eg. "%d min(s)". When i translate particular singular/plural i want to omit the part in parentheses (useless in our language), but then Transifex complain about missing  parentheses.

## Possible Solution
I do not know how the strings are implemented in sources. For now i do not translate these strings with plural form, because i afraid, that then all my translation is not saved at all.

**Comment by droidmonkey** [comment]
Please do omit the parentheses. I find this to be a bug in Transifex. In Transifex you can choose which form you are translating. I haven't seen Slovak, but there is probably three forms to choose.

TL;DR - ignore the warning trom Transifex

**Comment by slavkoja** [comment]
OK, i will try to ignore it ;-)

But when we talk about Transifex settings. Now i see, that Slovak is marked as "sk", but it doesn't matter. More important is, that Transifex provides four plural forms -- one, pair, more and others (raw translated, i see them in Slovak) and i don't see where i can change it. Can you help me with this, please?

**Comment by droidmonkey** [comment]
I cannot help you any further. I only have a fundamental understanding of Transifex and translating in general

**Comment by slavkoja** [comment]
OK, i will close it. I tried it, and it seems, that string is saved despite warning.

**Comment by phoerious** [comment]
The warning is a Transifex bug, just ignored it. The number of plurals should be defined in the language spec provided by Transifex. When I open the Slovak translation, I can see forms for 1, few, many, and others.

**Comment by slavkoja** [comment]
Yes, there are two plural forms in our language, one for 2-4, and one for 5 and more.

I am familiar with these settings, i am translating for years, but mostly PO (gettext) files, where i have to set plural forms in any PO file (of course, i have presets in my gtranslator). But i am not very familiar with TS files...

**Comment by slavkoja** [comment]
I took look into my other translation on Transifex, which use PO files and i never used Transifex for it, but i always upload my PO files.

The Transifex sets in it:
```
"Plural-Forms: nplurals=4; plural=(n % 1 == 0 && n == 1 ? 0 : n % 1 == 0 && n >= 2 && n <= 4 ? 1 : n % 1 != 0 ? 2: 3);\n"
```
This corresponds with our previous talk. But [gettext's manual](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html) tells (look for Slovak & Czech language):
```
Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;
```
The difference seems to be, that the Transifex's plural form is taking decimal numbers into account = the last (fourth) form is for numbers with decimals.
